README.md

```diff
@@ -7,32 +7,59 @@
 
 > Share your conferences activity to your buddies
 
+## Requirements
+
+This project requires:
+
+- Docker 
+- [mkcert](https://github.com/FiloSottile/mkcert)
+- [pipenv](https://github.com/pypa/pipenv)
+
 ## Installation
 
 ### Local
 
-> Add `local.starfleet.app` to your hosts file
-
-Use [mkcert](https://github.com/FiloSottile/mkcert) to install a CA on your system to be able to generate a valid self-signed certificate for local.starfleet.app
+First, you will need to add `local.starfleet.app` to your hosts file.
+Open it with
+```
+sudo nano /etc/hosts
+```
 
-`$ mkcert *.starfleet.app`
+Then paste in `127.0.0.1 local.starfleet.app`
 
-> certificate and key must be placed in infrastructure/docker/services/router/etc/ssl/certs
+You then need [mkcert](https://github.com/FiloSottile/mkcert) to generate a certificate for `local.starfleet.app`
 
-Install Python virtualenv to be able to use fabric commands to control easily the Docker stack
+When mkcert is installed, you need to install Python [pipenv](https://github.com/pypa/pipenv) to be able to use fabric commands to easily control the Docker stack.
```

Comment:
The workflow is a bit odd. You talk about mkdir, but then the command is about pipenv, 
So IMHO, you must put the mkcert sentence after the pipenv block